### ISSUE TITLE
New MEW website multiple networks support

### ISSUE THREAD

**Issue by Kryptoxic** [MyEtherWallet/MyEtherWallet#819]
- **I'm submitting a ...**

  - [x] Feature request
  - [ ] Bug report

- **Feature Request**

  - Briefly explain the feature you would like us to implement

Implement a network selection list similar to the vintage MEW.

  - What are the current issues this feature will solve

This will solve the need for using the vintage website to access altcoins like CLO or TOMO

  - Is this feature cross compatible with other ethereum networks

Yes

  - Are you willing to help us with development support if required

Yes but I have no prior knowledge but I am happy to test any beta or alpha versions.

  - Where can this be added? e.g: `/interface`, `/create-wallet`, etc.

/access-my-wallet

**Comment by gamalielhere** [comment]
@Kryptoxic This is already implemented. If you go access your wallet, you can click on 
<img width="368" alt="Screen Shot 2019-03-18 at 2 51 33 PM" src="https://user-images.githubusercontent.com/16236741/54566174-5a8e4380-498d-11e9-9bf6-6a3ea5150284.png"> the `change` button and that shows the supported networks on MEW.

**Comment by Kryptoxic** [comment]
> @Kryptoxic This is already implemented. If you go access your wallet, you can click on
> <img alt="Screen Shot 2019-03-18 at 2 51 33 PM" width="368" src="https://user-images.githubusercontent.com/16236741/54566174-5a8e4380-498d-11e9-9bf6-6a3ea5150284.png"> the `change` button and that shows the supported networks on MEW.

Could you screenshot the entire screen? I'm having difficulty finding the change button on the website.

**Comment by gamalielhere** [comment]
<img width="1920" alt="Screen Shot 2019-03-19 at 1 21 59 PM (2)" src="https://user-images.githubusercontent.com/16236741/54639185-0006ed00-4a4a-11e9-97f6-04e961a28823.png">

**Comment by gamalielhere** [comment]
<img width="1920" alt="Screen Shot 2019-03-19 at 1 22 33 PM (2)" src="https://user-images.githubusercontent.com/16236741/54639236-2036ac00-4a4a-11e9-8489-43a2f43ab733.png">

**Comment by Kryptoxic** [comment]
Okay now I know where to change the network, however, when I select the option to unlock my wallet using my ledger, it only allows me to log in with my ethereum app on my Ledger Nano S.

![image](https://user-images.githubusercontent.com/15937274/54659642-acf66e00-4b0d-11e9-9b43-b63628387709.png)

There is nowhere that I can change the network and the only way the website can detect my Ledger is when I select the Ethereum app on my ledger which then provides me the option to choose the network and derivation paths.

![image](https://user-images.githubusercontent.com/15937274/54659688-e202c080-4b0d-11e9-8813-35e6ca33e32e.png)

**Comment by gamalielhere** [comment]
@Kryptoxic you have to download the appropriate app from Ledger live first, then connect to the network you're trying to use

**Comment by Kryptoxic** [comment]
> @Kryptoxic you have to download the appropriate app from Ledger live first, then connect to the network you're trying to use

Yes, for example, I want to access Ubiq through MyEtherWallet and I have already installed the Ubiq app from Ledger Live. I have selected the app on my Ledger Nano S but MyEtherWallet refuses to open the screen where I can select the network. It keeps showing me this screen instead.

https://user-images.githubusercontent.com/15937274/54659642-acf66e00-4b0d-11e9-9b43-b63628387709.png
